zshrc-additions

```diff
@@ -0,0 +1,40 @@
+# The following sections can be added to the bottom of your ~/.zshrc file
+# They may provide improved integrations with certain CLI tools
+
+# nvm config
+export NVM_DIR="$HOME/.nvm"
```

Comment:
What are your thoughts on `oh-my-zsh`?  It has plugins that bake in all of these configurations.

Reply:
Great stuff! If there's anything here that is superseded by `oh-my-zsh`, I think it'd be worth removing from the `.zshrc` and pointing people to `oh-my-zsh`.

Reply:
never mind! still no auto nvm

Reply:
Actually... I just realized you already installed `oh-my-zsh` in the `mac` setup. In this case I recommend
```
plugins=(git gitfast nvm pyenv pipenv)
```

Reply:
`gitfast` is new to me, and looks great! 

I'll add those suggestions to the `zshrc-append` (and put them in my personal `.zshrc`).